### ISSUE TITLE
1.0.0 Feature List

### ISSUE THREAD

**Issue by jameslkingsley** [ARCOMM/ARCMC#1]
- [x] Terrain detail and view distance settings
- [ ] Entering Zeus on the fly as mission controller
- [ ] GAIA support
- [ ] Teleporting groups/units

**Comment by alexcroox** [comment]
- Could you make sure Garrison keeps units inside buildings (or make this optional) through disabling fleeing etc. A lot of Garrison functions put the units in the buildings but as soon as the firefight starts they run out to help, meaning when you arrive in town there is no CQB to do :(

- Place invisible spawn object for easy spawning of units at height, more detail: https://github.com/astruyk/Ares/issues/194

**Comment by jameslkingsley** [comment]
Yeah garrison will ensure units stand their ground. It will most likely be a toggle so you tell units to exit the garrison state.

As to the object height thing, I think with the control we have with this mod - being able to change literally anything in object movement, we could try to implement the surface detection that's done in Eden. If that fails, then perhaps an extra option in the asset browser to spawn as a "special" object for these sorts of things.

**Comment by alexcroox** [comment]
Surface detection would be incredible.

**Comment by cx64** [comment]
How about a 'defense in depth' Infantry Command? Units will defend until a certain percentage of casualties have been inflicted upon by the attacking force; units will then retreat a certain distance. Both percentage of casualties and retreating distance should be changeable on-the-fly, as well as having one of both be controlled by the mission curator (ie zeus, admin and game master).

**Comment by jameslkingsley** [comment]
I'm not sure. I feel like that can be controlled via the mission controller anyway. When the mission controller feels the AI should flee, he can simply tell them to flee.

**Comment by alexcroox** [comment]
I'd love a "scout from high ground" (terrible name) feature that works similar to the current Ares "search and garrison" in that AI walk into the nearest building. However instead I'd like them to head for roof and/or the top floor and look out windows. Real players would go for a height advantage over their enemy and multi level combat is often missing from coop missions unless the mission director manually places a unit up high or takes control and climbs stairs to start firing on players from the vantage point. 

We did a defend mission the other night and players loved it when I moved an AI to the top floors of a construction building and fired on them from there. Especially when they are only used to AI swarming in from ground level. I can manually place units in top floors but it's very time consuming on an individual AI basis.

**Comment by SchwererKonigstiger** [comment]
- The ability to teleport players to a regroup point (flagpole)
 - The ability to create "regroup zones" for AI, so if you give them a fall back command, they group up there. 
 - The ability to force AI to fall back; used with above.

**Comment by jameslkingsley** [comment]
The two fallback options are kind of redundant. There's already a "Force Move" command which makes the units run towards a position not caring about combat (much). So the fallback thing is pretty much just a case of telling some units to force move somewhere.
